### ISSUE TITLE
merging lists?

### ISSUE THREAD

**Issue by airtonix** [tanbro/pyyaml-include#46]
files:

```
0.yml
a/b/c
  1.yml
  2.yml
f/j/k
  4.yml
  5.yml
```

`{1,2,3,4}.yml`
```
name: 1 # or 2,3,4
```

`0.yml`
```yml
title: merged lists

items:
  - <<: !include "a/b/c/*.yml"
  - <<: !include "f/j/k/*.yml"
```

expected output: 

```
title: merged lists

items: 
  - name: 1
  - name: 2
  - name: 3
  - name: 4
```

variations I've tried: 

`0a.yml`
```yml
title: merged lists

x-one: !include &one "a/b/c/*.yml"
x-two: !include &two "f/j/k/*.yml"

items:
  - <<: *one
  - <<: *two
```

**Comment by tanbro** [comment]
The list added under a list is a sub-list

Consider we have such a YAML:

```yaml
items: !include *.yaml
```

if all files match `*.yaml` contain a **Sequence** at the top level in it, what parsed and loaded are:

```yaml
items: [
  [item, item, item, ...] # items of 1st file match *.yaml
  [item, item, item, ...] # items of 2nd file match *.yaml
  # ....
  [item, item, item, ...] # items of n-th file match *.yaml
  # ...
]
```

because YAML content of each matched file are treated as a list(sequence).

So i think, if a `flatten` function was provided, like:

```yaml
items: !include {urlpath: *.yaml, flattern: true}
```

we could get:

```yaml
items: [
  item 0 of 1st file, item 1 of 1st file, ... , item n of 1st file,  # ...
  item 0 of 2nd file, item 1 of 2nd file, ... , item n of 2nd file,  # ...
  # ....
  item 0 of n-th file, item 1 of n-th file, ... , item n of n-th file,  # ...
  # ...
]
```

**Comment by tanbro** [comment]
[feature/flattern-sequences-when-many-files-matched](https://github.com/tanbro/pyyaml-include/tree/feature/flattern-sequences-when-many-files-matched) is for the requirement.

It will be released later.

**Comment by JaredCorduan** [comment]
first of all, thank you @tanbro for this library!

Are you planning on making a new pypi release soon? I'm excited for this feature. :)

**Comment by tanbro** [comment]
> first of all, thank you @tanbro for this library!
> 
> Are you planning on making a new pypi release soon? I'm excited for this feature. :)

Now the feat released in v2.2, see <https://pypi.org/project/pyyaml-include/2.2/>
